source/doc_versions.py

```diff
@@ -1,9 +1,9 @@
 # The API version strings to display in the documentation
 # COMMON_VERSION is the latest version of any one API, and is considered the overall API version
-COMMON_VERSION = '2016-06-12'
+COMMON_VERSION = '2016-07-01'
```

Comment:
Well, this won't exactly impact or break anyone's code. If you set up your client to call invoca.com/.../**2016-05-01**/....json, then nothing will have changed for you. It's only once you call invoca.com/.../**2016-07-01**/....json, that there will be a change in behavior.

One could argue that, by not bumping the documentation, we are depriving api users of the knowledge about this new update. But since we aren't updating the documentation anyway, they wouldn't know about it either way.

Mike definitely has a point here.